tests/integration/daemon/distributed/test_simple_local_remote/test_integration.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+import pytest
+
+from jina import Document
+from jina.flow import Flow
+
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+compose_yml = os.path.join(cur_dir, 'docker-compose.yml')
+flow_yml = os.path.join(cur_dir, 'flow.yml')
+
+
+@pytest.mark.parametrize('docker_compose', [compose_yml], indirect=['docker_compose'])
+def test_flow(docker_compose, mocker):
+    text = 'cats rules'
+    m = mocker.Mock()
+
+    def validate_output(resp):
+        m()
+        assert len(resp.index.docs) == 1
+        assert resp.index.docs[0].text == text
+
+    os.environ['JINA_ENCODER_HOST'] = '172.28.1.1'
+    os.environ['JINA_INDEXER_HOST'] = '172.28.1.2'
```

Comment:
Is it possible to put these into the `.env`? This can make maintenance easier.

Reply:
no, because these env live locally with the Flow. so they are fixed by the docker compose definition